cmd/auth/main.go

```diff
@@ -20,217 +15,142 @@ import (
+	if err != nil {
+		log.Fatalf("failed to connect to keto gRPC: %s", err.Error())
 	}
-
-	tracer, closer, err := jconfig.Configuration{
-		ServiceName: svcName,
-		Sampler: &jconfig.SamplerConfig{
-			Type:  "const",
-			Param: 1,
-		},
-		Reporter: &jconfig.ReporterConfig{
-			LocalAgentHostPort: url,
-			LogSpans:           true,
-		},
-	}.NewTracer()
+	//create keto write grpc client
+	writerConn, _, err := grpcClient.Connect(grpcClient.Config{ClientTLS: false, URL: fmt.Sprintf("%s:%s", cfg.KetoWriteHost, cfg.KetoWritePort)})
 	if err != nil {
-		logger.Error(fmt.Sprintf("Failed to init Jaeger: %s", err))
-		os.Exit(1)
+		log.Fatalf("failed to connect to keto gRPC: %s", err.Error())
 	}
 
-	return tracer, closer
-}
+	svc := newService(db, dbTracer, cfg.Secret, logger, readerConn, writerConn, cfg.LoginDuration)
 
-func initKeto(hostReadAddress, readPort, hostWriteAddress, writePort string, logger logger.Logger) (readerConnection, writerConnection *grpc.ClientConn) {
-	readConn, err := grpc.Dial(fmt.Sprintf("%s:%s", hostReadAddress, readPort), grpc.WithInsecure())
+	// HTTP server
+	// create new http handler tracer
+	tracer, closer, err := jaegerClient.NewTracer("auth", cfg.JaegerURL)
 	if err != nil {
-		logger.Error(fmt.Sprintf("Failed to dial %s:%s for Keto Read Service: %s", hostReadAddress, readPort, err))
-		os.Exit(1)
+		log.Fatalf("failed to init Jaeger: %s", err.Error())
 	}
-
-	writeConn, err := grpc.Dial(fmt.Sprintf("%s:%s", hostWriteAddress, writePort), grpc.WithInsecure())
-	if err != nil {
-		logger.Error(fmt.Sprintf("Failed to dial %s:%s for Keto Write Service: %s", hostWriteAddress, writePort, err))
-		os.Exit(1)
+	defer closer.Close()
+	// create new http server config
+	httpServerConfig := server.Config{Port: defSvcHttpPort}
+	// load http server config from environment variables
+	if err := env.Parse(&httpServerConfig, env.Options{Prefix: envPrefixHttp, AltPrefix: envPrefix}); err != nil {
+		log.Fatalf("failed to load %s HTTP server configuration : %s", svcName, err.Error())
 	}
+	// create new http server
+	hs := httpserver.New(ctx, cancel, svcName, httpServerConfig, httpapi.MakeHandler(svc, tracer, logger), logger)
+
+	// gRPC server
+	// create new grpc server config
+	grpcServerConfig := server.Config{Port: defSvcGrpcPort}
+	// load grpc server config from environment variables
+	if err := env.Parse(&grpcServerConfig, env.Options{Prefix: envPrefixGrpc, AltPrefix: envPrefix}); err != nil {
+		log.Fatalf("failed to load %s gRPC server configuration : %s", svcName, err.Error())
+	}
+	registerAuthServiceServer := func(srv *grpc.Server) {
+		mainflux.RegisterAuthServiceServer(srv, grpcapi.NewServer(tracer, svc))
+	}
+	// create new grpc server
+	gs := grpcserver.New(ctx, cancel, svcName, grpcServerConfig, registerAuthServiceServer, logger)
 
-	return readConn, writeConn
-}
+	//Start servers
```

Comment:
Be consistent with comments (capitalize, full stop, spaces after `//`).

Reply:
done